central/telemetry/centralclient/interceptors_test.go

```diff
@@ -0,0 +1,102 @@
+package centralclient
+
+import (
+	"testing"
+
+	"github.com/stackrox/rox/generated/storage"
+	"github.com/stackrox/rox/pkg/set"
+	"github.com/stackrox/rox/pkg/telemetry/phonehome"
+	"github.com/stretchr/testify/assert"
+)
+
+func Test_clusterRegistered(t *testing.T) {
```

Comment:
I find these tests a little bit hard to follow with the number of asserts. Could you maybe add some comments to make the structure and purpose of the test more obvious?